### ISSUE TITLE
Cannot Add Portal in Survey123 1.8.21

### ISSUE THREAD

**Issue by jmirmelstein** [Esri/Survey123Community#578]
When attempting to connect the Survey123 app to a local Portal for ArcGIS, I am unable to Add a Portal from the Sign-In Page as the button is nonfunctional. 

Android 7.1

![screenshot_20161107-111736](https://cloud.githubusercontent.com/assets/3289025/20070033/2af6e56c-a4dc-11e6-90b1-8b245dadefb1.png)

**Comment by jmirmelstein** [comment]
@slibby

**Comment by bowlesgsrcx** [comment]
Can you tap on URL of your Portal for ArcGIS and enter the portal URL?

**Comment by jmirmelstein** [comment]
Yes, good catch.  So it's not a functional issue but just a UI issue.  

![screenshot_20161107-113111](https://cloud.githubusercontent.com/assets/3289025/20070691/a8b5b92c-a4de-11e6-8c03-8b52592a8e94.png)

**Comment by tedrick** [comment]
Thanks @jmirmelstein - I've put an issue in to look at rendering in Nougat.

**Comment by slibby** [comment]
@tedrick is your understanding that this is likely limited to that version, other versions work as expected?

**Comment by tedrick** [comment]
I haven't seen this behavior in other versions of android; given that 7.1 is very recently released (not even officially released yet), I'm assuming it's an issue specific to that version

**Comment by jmirmelstein** [comment]
@tedrick I would be curious if this also exists in 7.0.  The new google Pixel ships directly with 7.1 preinstalled, so while it might not be 'released', I'm not sure that it's Beta either.
